packages/angular/cli/commands/update-impl.ts

```diff
@@ -177,6 +179,8 @@ export class UpdateCommand extends Command<UpdateCommandSchema> {
           }
         }
 
+        this.logger.error('⨉  Migration failed. See above for further details.\n');
```

Comment:
You can use `colors.symbols.cross` to ensure a cross platform `x`

The same with the others.  `colors.symbols.pointer`/`colors.symbols.check`